### PR TITLE
PDF for bert

### PR DESCRIPTION
**Pull request by steveoh**
skeleton

### CODE REVIEW

File: src/pages/products/sgid/cadastre/parcels.astro [hunk 1]
```diff
@@ -19,6 +19,8 @@ import GridForMoreResources from '@components/page/GridForMoreResources.astro';
 import GridForYouMightLike from '@components/page/GridForYouMightLike.astro';
 import QuickLinks from '@components/page/QuickLinks.astro';
 import Section from '@components/page/Section.astro';
+import download from '@downloads/product-sgid/cadastre/ExpandedParcelDataSharingImplementationGuidelinesDRAFT.pdf?url';
+import download from '@downloads/product-sgid/cadastre/Working Group Recommendations 5.19.16.docx?url';
```
Comment (steveoh) [inline src/pages/products/sgid/cadastre/parcels.astro:23]: these are variables and need to be uniquely named.
```suggestion
import recommendationDocument from '@downloads/product-sgid/cadastre/Working Group Recommendations 5.19.16.docx?url';
```

File: src/pages/products/sgid/cadastre/parcels.astro [hunk 2]
```diff
@@ -386,10 +388,11 @@ const countyMetadata = [
...
       development, transportation, planning and the provisions of public services. In May of 2016, the Land Information
-      Records work group released its <ExternalLink
-        href="https://drive.google.com/file/d/1E3ks5ndjMKiZqS5b7N6a3_1paJK0jHi_"
-        >Land Use Information Records Recommendations</ExternalLink
-      >.
+      Records work group released its 
+     <a href={downloads/product-sgid/cadastre/Working Group Recommendations 5.19.16.docx}>Land Use Information Records Recommendations
+    </a> and 
+    <a href={downloads/product-sgid/cadastre/ExpandedParcelDataSharingImplementationGuidelinesDRAFT.pdf}
+      >Expanded Parcel Data Sharing Implementation Guidelines</a>.
```
Comment (steveoh) [inline src/pages/products/sgid/cadastre/parcels.astro:395]: then you can use the variable as the value.
```suggestion
     <a href={recommendationDocument}>Land Use Information Records Recommendations
    </a> and 
    <a href={download}>Expanded Parcel Data Sharing Implementation Guidelines</a>.
```

**Review by steveoh — COMMENTED** [review]
pretty close! Don't forget to delete the keep file.

**Comment by rkelson** [comment]
this should be ready to go

**Pull request merged**